### PR TITLE
New translations into Spanish

### PR DESCRIPTION
**Pull request by diegolgz**
Translated into Spanish:
* Alias strings
* Loops
* Foreach
* Ranges
* Associative arrays

Also added some non-translated files (under gems) to avoid link errors.

### CODE REVIEW

**Comment by diegolgz** [comment]
One question: when will these translations be available in main D Tour site?

**Comment by wilzbach** [comment]
In theory there's an auto-deploy setup that deploys everything automatically (takes ~10 after a PR).
Though it seems that there are currently some permanent errors with ldc that prevented the deploy :/

https://travis-ci.org/dlang-tour/core

I will look into them - thanks for the ping!

File: basics/alias-strings.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+También es posible usar *raw strings* (cadenas sin procesar) para minimar la
+laboriosa tarea de *escapar* (*escape*) símbolos reservados. Las cadenas sin
+procesar pueden ser declaradas o bien usando acentos graves —*backticks* en
+inglés— (`` ` ... ` ``) o bien usando el prefijo `r` —de *raw*— (`r" ... "`).
+
+    string raw  =  `raw "string"`; // cadena sin procesar
+    string raw2 = r"raw `string`"; // cadena sin procesar
+
+D provides even more ways to represent strings - don't hesitate
```
Comment (csantander) [inline basics/alias-strings.md]: Needs translation

File: basics/associative-arrays.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# Arrays asociativos
+
+D arrays asociativos incluidos en el propio lenguaje. Estos arrays asociativos
```
Comment (csantander) [inline basics/associative-arrays.md]: La primera oración necesita ser revisada

**Comment by wilzbach** [comment]
> I will look into them - thanks for the ping!

https://github.com/dlang-tour/english/pull/247

**Pull request merged**